lib/iris/fileformats/grib/message.py

```diff
@@ -60,7 +60,7 @@ def messages_from_filename(filename):
                     break
                 raw_message = _RawGribMessage(grib_id)
                 recreate_raw = _MessageLocation(filename, offset)
-                yield _GribMessage(raw_message, recreate_raw)
+                yield GribMessage(raw_message, recreate_raw)
 
     def __init__(self, raw_message, recreate_raw):
         """
```

Comment:
Also needed (but outside the scope of the current diffs):
The 'GribMessage.sections' property should have a docstring explaining that it is a **_mapping_**
 i.e.  " int ::--> :class:`Section` "
A quick usage example should be worth a lot of prose there, I think.